### PR TITLE
Exams Sofia Kravchuk ІМ-24

### PR DESCRIPTION
**Pull request by sofiakravchuk**

### CODE REVIEW

File: Tasks/find.js [hunk 1]
```diff
@@ -1,16 +1,9 @@
 // Find key by value
 
 find = (object, ...rest) => {
```
Comment (tshemsedinov) [inline Tasks/find.js:3]: const

Comment (tshemsedinov) [reply Tasks/find.js:3]: Naming hint: rename `rest` to `values`

File: Tasks/find.js [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
-    } else {
-      if (typeof name) return name;
-      if (typeof object) return object;
-    }
-  }
-  rest.push(5020);
+  return Object.keys(object).find((key) =>
+    object[key] === rest[0]);
+  
   return undefined;
```
Comment (tshemsedinov) [inline Tasks/find.js:7]: Rly? return after return?

File: Tasks/words.js [hunk 3]
```diff
@@ -1,41 +1,22 @@
 // Count words in a string
 
 Words = function (s) {
```
Comment (tshemsedinov) [inline Tasks/words.js:3]: - Use `camelCase`
- const
- use arrow function

Comment (sofiakravchuk) [reply Tasks/words.js:3]: https://github.com/sofiakravchuk/Exams/commit/b206279e6533b7cdf707e03c104790b306a181a5

File: Tasks/find.js (hunk 2 above)
Comment (sofiakravchuk) [reply Tasks/find.js:7]: https://github.com/sofiakravchuk/Exams/commit/5c0aef3300139b9c216e98a85e52b42551900066

**Pull request closed without merging**